Add a post about invasive ad tech and publishers (#150)
readthedocs/ethicalads.io: content/posts/2021-invasive-ad-targeting-bad-journalism-premium-publishers.md

Title: How Invasive Ad Targeting is Bad for Journalism and Other High-Quality Publishers
Date: May 25, 2021

> description: The lie from ad tech is that there's no such thing as a premium publisher and all ad views are basically the same.

Reviewer: This feels strongly worded to me, but maybe we need to do that :)

Author: I think we should go with it!

> tags: cookies

Reviewer: ```suggestion
tags: ad-tech, publishers
```

authors: David Fischer
image: /images/posts/2021-invasive-ad-targeting.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@jeswinthomas?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Jeswin Thomas</a> on <a href="https://unsplash.com/s/photos/watching?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>

> It used to be if you wanted to reach a particular group of people with your message
> you’d buy ad space on a billboard where those people are likely to see it
> or buy an ad spot in a newspaper or during a TV show that your target market likes.

Reviewer: ```suggestion
It used to be if you wanted to reach a particular group of people with your message
you’d buy ad space on a billboard where those people are likely to see it
or buy an ad spot in a newspaper section that your target market reads.
```

Feels simpler without 2 or's.

In the online world, this used to mean finding an online publication for your product
or buying up specific terms at search engines.
If you plan to introduce a new line of cat food?
You run an ad on [ModernCat.com](https://moderncat.com)
or you pay to take the top search engine result for "cat food".

With the rise of ad tech powered by personal data collection,
advertisers can reach cat lovers (or any targeted group)
by paying ad tech companies to show their ads wherever the audience goes.
Maybe you pay top dollar for ModernCat.com for a little while,
but after you've got cookies to track their visitors onto other sites,

> you can reach the same folks on lower value publications more cheaply.

Reviewer: ```suggestion
you can reach the same folks on lower value channels more cheaply.
```

Can you target across devices? If not, I think we probably want `websites` or similar here vs. publications.

## What impression commoditization means to publishers

> The result of this

Reviewer: define this?

is that advertising dollars have moved from publications with well researched content written by knowledgeable authors
to low quality SEO optimized sites with re-purposed material developed to maximize paid ad impressions.

> While there are a lot of reasons for the fall of newspapers, this dynamic is a major factor.

Reviewer: Also craigslist! 👿

Instead of spending money on investigative reporters,
it makes more sense to churn out clickbait that rephrases research done somewhere else.

> In essence, all web-views are the same from an ad tech-powered advertising perspective.

Reviewer: This sentence could be simplified..somehow. Perhaps:

```suggestion
In essence, all impressions are the same from an ad tech-powered perspective.
```

Also not sure if we have standard words here, but maybe `impressions`? We use `Ad views` below... /shrug

Ad views have become a commodity.
For most ad-funded sites including many newspapers,
they now compete with every other web site in a race to attract the most views rather than focusing on quality.

## What publishers can do about it

Publishers can take back their audience and advertising revenue in two ways.

> Cut off ad tech surveillance technology and stop providing ad space for these companies to show ads.

Reviewer: Should these be bullets? Perhaps bolded? It feels like we need to make it pop a bit more.

Author: I made them bullets with bold!

Title: How Invasive Ad Targeting is Bad for Journalism and Other High-Quality Publishers
Date: May 25, 2021

> description: The lie from ad tech is that there's no such thing as a premium publisher and all ad views are basically the same.

Reviewer 2: ```suggestion
description: Premium publishers exist, some views are more viable than others, and why ad tech companies lie about it.
```
I don't have a problem with the strong wording, I just think we should spend more focus on what we believe rather than what they believe.

Author: I want to try a more negative description since this will be displayed by search engines under the headline of the post:

    How Invasive Ad Targeting is Bad for Journalism and Other High-Quality Publishers
    The lie from ad tech is that there's no such thing as a premium publisher and all ad views are basically the same.

tags: ad-tech, publishers
authors: David Fischer
image: /images/posts/2021-invasive-ad-targeting.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@jeswinthomas?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Jeswin Thomas</a> on <a href="https://unsplash.com/s/photos/watching?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>

It used to be if you wanted to reach a particular group of people with your message
you’d buy ad space on a billboard where those people are likely to see it
or buy an ad spot in a newspaper section that your target market reads.

> In the online world, this used to mean finding an online publication for your product

Reviewer 2: ```suggestion
In the online world, this translated into finding an online publication for your product
```

Author: I think it's important to talk about how things were and how they've become. I'm not sure just using the past tense is enough here.

Reviewer 2: I just didn't like "used to" being used twice in two sentences. How about "Before the internet" for the first sentence and then keeping "used to mean" for the second?

Author: Fair. How about, "Before micro-targeted advertising, this translated into..."

> or buying up specific terms at search engines.

Reviewer 2: ```suggestion
or buying up specific, relevant terms on search engines.
```

If you plan to introduce a new line of cat food?
You run an ad on [ModernCat.com](https://moderncat.com)
or you pay to take the top search engine result for "cat food".

With the rise of ad tech powered by personal data collection,
advertisers can reach cat lovers (or any targeted group)
by paying ad tech companies to show their ads wherever the audience goes.
Maybe you pay top dollar for ModernCat.com for a little while,
but after you've got cookies to track their visitors onto other sites,

> you can reach the same folks on lower value channels more cheaply.

Reviewer 2: Lil vague to the naive reader. What's a lower value channel? Cheaper? Less relevant?

Author: Probably all of the above. Cheaper definitely but lower quality.

## What impression commoditization means to publishers

> The result of this intrusive targeting is a race the bottom.

Reviewer 2: Haven't yet explained where the intrusion is in this piece. I like the race to the bottom though.

Author: I updated it slightly.

> Advertising dollars have moved from publications with well researched content written by knowledgeable authors

Reviewer 2: ```suggestion
Advertising dollars have moved away from publications with well researched content written by knowledgeable authors
```

to low quality SEO optimized sites with re-purposed material developed to maximize paid ad impressions.

> While there are a lot of reasons for the fall of newspapers, this dynamic is a major factor.

Reviewer 2: ```suggestion
While there are a lot of reasons for the fall of newspapers, this dynamic shift is a major factor.
```

Instead of spending money on investigative reporters,

> it makes more sense to churn out clickbait that rephrases research done somewhere else.

Reviewer 2: ```suggestion
it makes more economic sense to churn out clickbait that rephrases research done elsewhere.
```

> In essence, all impressions are the same from an ad tech perspective.

Reviewer 2: How did you draw this conclusion from the proceeding facts?

Author: I guess the answer is that it doesn't matter "where ads appear" if you're getting mostly the same people. I'm not sure I need to be that explicit. Maybe I do.

Author: I made it more explicit.

Ad views have become a commodity.
For most ad-funded sites including many newspapers,

> they now compete with every other web site to attract the most views rather than focusing on quality.

Reviewer 2: ```suggestion
the race for views comes at the cost of sacrificing quality.
```

I'm reminded of those "breaking news" stubs that are published before anyone has any real information...

Publishers can take back their audience and advertising revenue in a few ways.

* **Cut off ad tech surveillance technology.**
  Ad tech is driven by personal data which, with some exceptions, is collected from individual publishers sites
  through the use of third-party cookies, fingerprinting, and other tracking technologies.
  Publishers should eliminate or severely restrict what ad tech can run on their site.
* **Stop providing ad space for tracking companies to show ads.**
  Where it isn't possible to restrict the ad network in terms of tracking,
  remove ads that track users completely.

This doesn't mean giving up your ad revenue, though.
While there seems to be unlimited ad space online there is a very limited amount of quality ad space.
Larger publishers can work directly with advertisers to privately place ads often driving up their own revenue
[as the New York Times did](https://open.nytimes.com/to-serve-better-ads-we-built-our-own-data-program-c5e039bf247b).
For publishers that are too small or unable to hire a sales force,
they can find an ad network that will place ads without sacrificing user privacy
or siphoning off their audience's data for the ad network's gain.

> This is the reason we built the EthicalAds network.

Reviewer 2: What is the antecedent of "this"?

Author: I'll update it to "Giving small and mid-size publishers a way to monetize their audience without giving up their privacy is the reason..."

There are lots of developer-focused sites that are too small to sell ads directly.
If you're looking for an ad network that won't compromise your audience,

> [join EthicalAds]({filename}../pages/publishers.md) and help preserve privacy on the web!

Reviewer 2: Love this call to action

## What impression commoditization means to publishers

The result of this intrusive targeting is a race the bottom.
Advertising dollars have moved from publications with well researched content written by knowledgeable authors
to low quality SEO optimized sites with re-purposed material developed to maximize paid ad impressions.
While there are a lot of reasons for the fall of newspapers, this dynamic is a major factor.
Instead of spending money on investigative reporters,
it makes more sense to churn out clickbait that rephrases research done somewhere else.

In essence, all impressions are the same from an ad tech perspective.
Ad views have become a commodity.
For most ad-funded sites including many newspapers,
they now compete with every other web site to attract the most views rather than focusing on quality.

## What publishers can do about it

Publishers can take back their audience and advertising revenue in a few ways.

* **Cut off ad tech surveillance technology.**
  Ad tech is driven by personal data which, with some exceptions, is collected from individual publishers sites
  through the use of third-party cookies, scripts, and other technologies.

> * **Stop providing ad space for these companies to show ads.**

Reviewer: We should probably define `these technologies` in at least one of the uses in this bullet

Author: Third-party cookies is probably the big one. You think we need to be more precise about how that works? That is sort of covered in the first section (wrt moderncat.com)

Reviewer: I just think users will have no idea what technologies we're talking about.

Author: Ok, to make this concrete:

* Change the above line "through the use of third-party cookies, scripts, and other technologies." to "through the use of third-party cookies, fingerprinting, and other tracking technologies."
* Change the line below from "eliminate or severely restrict the use of these technologies and the companies that use them." to "eliminate or severely restrict what ad-tech can run on their site in terms of tracking, fingerprinting, and third-party cookies. Where that isn't possible, remove ads that track users completely."

Reviewer: Sounds good 👍

Author: It does feel slightly repetitive as I read it. Maybe I'm overthinking but we could go to:

    * **Cut off ad tech surveillance technology.**
      Ad tech is driven by personal data which, with some exceptions, is collected from individual publishers sites
      through the use of third-party cookies, fingerprinting, and other tracking technologies.
      Publishers should eliminate or severely restrict what ad-tech can run on their site
    * **Stop providing ad space for these companies to show ads.**
      Where it isn't possible to restrict the ad network in terms of tracking,
      remove ads that track users completely.

Reviewer: I just want each bullet to stand on it's own, rather than referencing something else.

Author: I changed it one more time.
